docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,246 @@
+---
+sidebar_label: 'Upgrade LSP6 Key Manager'
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade the LSP6 Key Manager
+
+:::info Requirements
+
+You will need a Universal Profile that you can control via its KeyManager to follow this guide.
```

Comment:
this callout can be deleted imo

Reply:
I feel like it is a requirement in order to follow the guide 🤔